.github/workflows/reusable-rust-test.yml

```diff
@@ -57,11 +62,161 @@ jobs:
 
       - name: Run Cargo Check (all strategy)
         if: ${{ inputs.cargo-check-strategy == 'all' }}
-        run: cargo check --workspace --all-targets --locked
+        shell: bash
+        run: |
+          # Temporarily disable nightly flags for stable Rust cargo check
```

Comment:
Okay. I see your strategy here. But this is way too error-prone. Try to create a `.cargo/config.stable.toml` for CI test. You may use this file for CI checks.

Reply:
Looks like the CI uses the merged version of config.toml. Maybe you need to rename config.stable.toml to config.toml to fully fix this on CI.